File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/models/MB_S_22.java
package com.lenta.lentabp10.MobileObjectsService.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

// Модель справочника списка наборов
public class MB_S_22 {
    /// Номер товара
    @JsonProperty("MATNR_OSN")
    @Getter private String matnr_osn;

    /// Компонент спецификации
    @JsonProperty("MATNR")
    @Getter private String matnr;

    /// Количество вложенного
    @JsonProperty("MENGE")
    @Getter private double menge;

    /// Базисная единица измерения
    @JsonProperty("MEINS")
    @Getter private String meins;

    public MB_S_22(String matnr_osn, String matnr, double menge, String meins) {
        this.matnr_osn = matnr_osn;
        this.matnr = matnr;
        this.menge = menge;
        this.meins = meins;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/models/MB_S_22.kt
package com.lenta.bp10.mobileObjectsService.models

import com.google.gson.annotations.SerializedName

// Модель справочника списка наборов
class MB_S_22(
        /// Номер товара
        @SerializedName("MATNR_OSN")
        val matnr_osn: String,

        /// Компонент спецификации
        @SerializedName("MATNR")
        val matnr: String,

        /// Количество вложенного
        @SerializedName("MENGE")
        val menge: Double,

        /// Базисная единица измерения
        @SerializedName("MEINS")
        val meins: String
)